### PR TITLE
README: Add simple installation instructions

### PR DESCRIPTION
**Pull request by sils**
Closes https://github.com/coala-analyzer/coala-vim/issues/3
Closes https://github.com/coala-analyzer/coala-vim/issues/2

### CODE REVIEW

File: plugin/coala.vim [hunk 1]
```diff
@@ -1,40 +1,40 @@
 " g:linters_automatic_on_save is a global flag for enabling/disabling the
```
Comment (Adrianzatreanu) [inline plugin/coala.vim:1]: why exactly was there a problem with using tabs?

Comment (sils) [inline plugin/coala.vim:1]: we use spaces everywhere in coala and my vim is configured to only use spaces :P project policy? It's easier if people don't have to switch for this project

**Comment by Adrianzatreanu** [comment]
ack 5373758

**Comment by sils** [comment]
ack 5373758

**Comment by Adrianzatreanu** [comment]
LMAO

**Comment by Adrianzatreanu** [comment]
ack 29c0659

**Comment by Adrianzatreanu** [comment]
ack 32f3bef

File: plugin/coala.vim [hunk 2]
```diff
@@ -26,7 +26,7 @@ function s:RunLinter()
 	let l:temp_file = tempname()
 	set cmdheight=2
 	echo "Running coala... When this finishes, you will see a popup if there are any errors."
-	silent execute "!cd ".s:wd." && coala-json --find-config --limit-files=\"".expand('%:p')."\" | python ".s:filename."\/jsonp.py > " . l:temp_file
+	silent execute "!cd ".s:wd." && coala-json --find-config --limit-files=\"".expand('%:p')."\" | python2 ".s:filename."\/jsonp.py > " . l:temp_file
 	if v:shell_error
```
Comment (Adrianzatreanu) [inline plugin/coala.vim]: why arent we using python3?

File: plugin/coala.vim [hunk 3]
```diff
@@ -63,5 +63,3 @@ function! coala#run()
 endfunction
 
 au BufWritePost * call s:WritePostHook()
```
Comment (Adrianzatreanu) [inline plugin/coala.vim:65]: don't we leave a newline at the end of the files? i thought thats how pep8 works.

File: README.md [hunk 4]
```diff
@@ -8,6 +8,11 @@ This is a vim/gvim extension for coala.
 Simply save your file after loading it and you'll have coala running
 automatically and giving you the output in a popup.
 
+State
+=====
+
+This plugin is rather young. It is usable but beware of bugs! And file them.
```
Comment (Adrianzatreanu) [inline README.md]: I've seen `please` being used quite often in this situation. imo you should use : `And please file them!`

File: plugin/coala.vim (hunk 2 above)
Comment (sils) [inline plugin/coala.vim]: his script uses python2

File: plugin/coala.vim (hunk 3 above)
Comment (sils) [inline plugin/coala.vim:65]: yes there's one, it's usually not shown. We don't want two!

**Comment by Adrianzatreanu** [comment]
ack fad0605

**Comment by Adrianzatreanu** [comment]
ack 14b2d7a

**Comment by Adrianzatreanu** [comment]
ack ad3ecae

**Comment by Adrianzatreanu** [comment]
ack 9fbe2cf

**Pull request merged**